### ISSUE TITLE
Enhancing the Awareness, Utility, and Anonymity Set of Oxen - Part 3 - Promote ONS Purchasing by Invisible Social Graph Exploitation and Interactive In-App Prompts

### ISSUE THREAD

**Issue by venezuela01** [loki-project/loki-improvement-protocols#42]
# Enhancing the Awareness, Utility, and Anonymity Set of Oxen - Part 3 - Promote ONS Purchasing by Invisible Social Graph Exploitation and Interactive In-App Prompts

The [ORC-8](https://github.com/oxen-io/oxen-improvement-proposals/issues/36#issuecomment-1651732483) discussion identifies a pressing issue within the Oxen ecosystem: underutilization of the coin due to a limited number of daily transfers, thereby decreasing Oxen's practical anonymity set.

Using the [AARRR](https://builtin.com/growth-hacking/aarrr) framework as a foundation, we've outlined initial strategies in:

* [Enhancing the Awareness, Utility, and Anonymity Set of Oxen - Part 1 - Web Traffic Optimization](https://github.com/oxen-io/oxen-improvement-proposals/issues/40), and
* [Enhancing the Awareness, Utility, and Anonymity Set of Oxen - Part 2 - Oxen Airdrop for ONS (Session User Name)](https://github.com/oxen-io/oxen-improvement-proposals/issues/41).

These strategies aim to stimulate 'Acquisition', 'Activation', and potentially 'Referral'.

In this proposal, we introduce a **viral marketing strategy** that capitalizes on the **builtin and invisible social network** among Session users, further increase 'Referral'. The plan involves **interactive prompts** within the Session user interface to educate and encourage users to acquire their own ONS.

Imagine the following scenario: Alice uses Bob’s ONS to add him on Session. At this moment, Alice's Session UI should generate a prompt saying, **"Bob is using a Session User Name (ONS). If you'd like to purchase your own ONS, consider visiting these recommended locations to acquire some Oxen, or seek advice from your contacts."** This instant, interactive reminder efficiently guides Alice towards owning her ONS.

Simultaneously, when Bob accepts Alice's message request, his Session UI should alert him, **"You are being contacted by Alice through your ONS. Consider asking Alice if she's interested in obtaining her own ONS and if she needs any assistance."** This reminder encourages Bob to aid Alice's transition to becoming an Oxen user. Consequently, Bob might assist Alice in purchasing some Oxen from an exchange, selling some Oxen to Alice directly, or even gifting some Oxen to Alice.

In [Part 2 - Oxen Airdrop for ONS (Session User Name)](https://github.com/oxen-io/oxen-improvement-proposals/issues/41), we proposed dropping extra Oxen to users, providing them with additional resources to share with their close contacts, thereby increasing Oxen awareness and usage. This idea can be reinforced with our Interactive In-App Prompts strategy, creating a ripple effect starting from our airdrop 'seed' users that spreads to surrounding users.

**This viral marketing strategy is safe only with a privacy coin like Oxen, which can protect user privacy during transactions**. Without this, identities risk being de-anonymized due to the connections formed through transactions.

The precise phrasing and UI display of these prompts necessitates meticulous design by the marketing and product teams. It is essential to ensure a non-aggressive user experience, succinct and effective communication, and to preclude any misunderstandings regarding privacy concerns.

The technology underpinning the Oxen coin is impressive, yet its marketing potential remains underesitmated. We shouldn't rest solely on the product's technological strengths. Instead, we should develop innovative marketing strategies that leverage our deep understanding of the product's unique features, the invisible user relationships, and the uncovered potential of UI interaction.

**Comment by KeeJef** [comment]
Yeah, i would like to develop ONS functionality further than it has been thus far, i see a few options to further increase ONS uptake 

- In app ONS purchases 
- Special display of ONS name (name badge/profile changes etc...)
- Allow communities to target permissions based on ONS (only ONS can write messages, or access specific communities etc..)
- As you have stated, information displayed to users when they add someone via ONS or inspect an ONS name

I'm not sure if all of these are good ideas, but just some of my initial thinking around increasing usage of ONS.

**Comment by KeeJef** [comment]
> Yeah, i would like to develop ONS functionality further than it has been thus far, i see a few options to further increase ONS uptake
> 
> * In app ONS purchases
> * Special display of ONS name (name badge/profile changes etc...)
> * Allow communities to target permissions based on ONS (only ONS can write messages, or access specific communities etc..)
> * As you have stated, information displayed to users when they add someone via ONS or inspect an ONS name
> 
> I'm not sure if all of these are good ideas, but just some of my initial thinking around increasing usage of ONS.

**Comment by venezuela01** [comment]
Thanks for comments!

> Special display of ONS name (name badge/profile changes etc...)

Can you explain a bit more about this? I'm very interested in the idea and might spend some more time to explore this idea further.
